solutions/swb-reference/src/proxyHandlerLambda.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import axios from 'axios';
+
+/* eslint-disable-next-line */
+export async function handler(event: any) {
+  const URL = `${process.env.API_GW_URL!}/`;
+  let HTTP_METHOD = '';
+
+  const headers: { [id: string]: string } = {};
+  const response: { [id: string]: unknown } = {};
```

Comment:
Can we be more description with the type definition? It looks like there's aways four values. `statusCode`, `isBase64Encoded`, `body`, and `headers`.

We can also set `isBase64Encoded` to be false here. We can also preset the `headers` value. Those values are the same accross all `HTTP_METHOD`